Drift/Private/DriftBase.h

```diff
@@ -66,124 +66,127 @@ class FDriftBase : public IDriftAPI, public FTickableGameObject
     virtual ~FDriftBase();
 
     // FTickableGameObject API
-    void Tick(float DeltaTime) override;
+    virtual void Tick(float DeltaTime) override;
```

Comment:
UE4 uses meaningless virtual specifiers for overridden functions, but I argue it doesn't add to readability, and IDEs will flag it as not needed.

Reply:
I suppose the fact that Rider flags its absence by default is for consistency with UE4 code.  I found the setting for it and switched it around.